js/gated-content/src/views/BlogPage.vue

```diff
@@ -5,10 +5,13 @@
     </div>
     <div v-else-if="error">Error loading</div>
     <template v-else>
-      <div class="blog-page__image" v-bind:style="{
-              backgroundImage: `url(${image})`
-      }">
+      <div v-if="blog.attributes.field_vy_blog_image !== null">
+        <div class="blog-page__image" v-bind:style="{
+                backgroundImage: `url(${image})`
+        }">
+        </div>
       </div>
+      <div v-else><div class="blog-page__no-image"></div></div>
```

Comment:
Why is an empty div needed here? I tested on the sandbox and `div.blog-page__image` being gone seems sufficient to fix the formatting.